src/Bridge/Symfony/Bundle/Resources/views/DataCollector/request.html.twig

```diff
@@ -86,22 +93,26 @@
         <table>
             <thead>
                 <tr>
-                    <th>
+                    <th scope="col">
                         Filters
                     </th>
                 </tr>
             </thead>
-
             <tbody>
-                {% if collector.resourceMetadata.attributes.filters is defined %}
-                    {% for filter in collector.resourceMetadata.attributes.filters %}
-                        <tr>
-                            <td>
-                                {{ filter }}
-                            </td>
-                        </tr>
-                    {% endfor %}
-                {% endif %}
+                {% for id, filter in collector.filters %}
+                    <tr{% if filter is same as(null) %} class="status-warning"{% endif %}>
+                        <td>
+                            {{ id }}
+                            {% if filter is same as(null) %}<span class="newline text-muted">ignored filter</span>{% else %}{{ dump(filter) }}{% endif %}
```

Comment:
This condition is copy-pasted above. You could bind it to a variable.
Also, make it multiline.

Reply:
thanks @IonBazan, it has been updated